source/github-helpers/index.ts

```diff
@@ -52,11 +52,12 @@ export const buildRepoURL = (...pathParts: Array<string | number> & {0: string})
 };
 
 export const getPRHeadRepo = (): ReturnType<typeof getRepo> => {
-	return getRepo(select<HTMLAnchorElement>('.commit-ref.head-ref a')!);
+	const headLink = select('.commit-ref.head-ref a');
+	return getRepo(headLink);
```

Comment:
XO was complaining about the ! on GHA, but TypeScript wouldn't accept it without. This way it works for both. Don't ask me why